### PR TITLE
Added simple testing with mocha, chai and sinon

### PR DESCRIPTION
**Pull request by vermiculite**

### CODE REVIEW

File: test/setup.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var sinon = require('sinon');
+var chai = require('chai');
```
Comment (Mokto) [inline test/setup.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/theo-mathieu1/react-native-meteor/file/2383447738/issues/source?bid=3208427&fileBranchId=3226891#l2)

File: test/setup.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+var sinon = require('sinon');
```
Comment (Mokto) [inline test/setup.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/theo-mathieu1/react-native-meteor/file/2383447738/issues/source?bid=3208427&fileBranchId=3226891#l1)

File: test/setup.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+var chai = require('chai');
+var should = chai.should();
+var mockWebSocket = require('mock-socket').WebSocket;
```
Comment (Mokto) [inline test/setup.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/theo-mathieu1/react-native-meteor/file/2383447738/issues/source?bid=3208427&fileBranchId=3226891#l4)

File: test/setup.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+var mockServer = require('mock-socket').Server;
```
Comment (Mokto) [inline test/setup.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/theo-mathieu1/react-native-meteor/file/2383447738/issues/source?bid=3208427&fileBranchId=3226891#l5)

File: test/setup.js (hunk 1 above)
Comment (vermiculite) [inline test/setup.js]: :D

**Comment by Mokto** [comment]
Thanks !

**Pull request merged**